src/test/java/com/aws/greengrass/logmanager/LogManagerServiceTest.java

```diff
@@ -588,15 +589,17 @@ void GIVEN_cloud_watch_attempt_handler_WHEN_attempt_completes_THEN_successfully_
         CloudWatchAttempt attempt = new CloudWatchAttempt();
         Map<String, CloudWatchAttemptLogInformation> logStreamsToLogInformationMap = new HashMap<>();
         File file1 = new File(getClass().getResource("testlogs2.log").toURI());
+        String filehash1 = "123456";
         File file2 = new File(getClass().getResource("testlogs1.log").toURI());
+        String filehash2 = "987654";
```

Comment:
this definitely isn't the hash value for this log file, so why is this being used?